test/quantization/test_numeric_suite_fx.py

```diff
@@ -214,6 +243,77 @@ def compare_and_validate_results(float_model, q_model, module_swap_list, data):
             backup_prepared_model, q_model, module_swap_list, linear_data
         )
 
+    @override_qengines
+    def test_compare_model_stub_linear_dynamic_fx(self):
+        r"""Compare the output of dynamic quantized linear layer and its float shadow module"""
+
+        def compare_and_validate_results(float_model, q_model, module_swap_list, data):
+            ob_dict = compare_model_stub_fx(
+                float_model, q_model, module_swap_list, data
+            )
+            self.assertEqual(len(ob_dict), 1)
+            for k, v in ob_dict.items():
+                self.assertTrue(len(v["float"]) == len(v["quantized"]))
+                for i, val in enumerate(v["quantized"]):
+                    self.assertTrue(v["float"][i].shape == v["quantized"][i].shape)
+
+        float_model = SingleLayerLinearDynamicModel()
+        float_model.eval()
+
+        qconfig = torch.quantization.qconfig.default_dynamic_qconfig
+        qconfig_dict = {"": qconfig}
+
+        prepared_model = prepare_fx(float_model, qconfig_dict)
+
+        backup_prepared_model = copy.deepcopy(prepared_model)
+        backup_prepared_model.eval()
+
+        q_model = convert_fx(prepared_model)
+
+        linear_data = self.calib_data[0][0]
+        module_swap_list = [nn.Linear]
+
+        compare_and_validate_results(
+            backup_prepared_model, q_model, module_swap_list, linear_data
+        )
+
+    @override_qengines
+    def test_compare_model_stub_lstm_dynamic_fx(self):
+        r"""Compare the output of dynamic quantized linear layer and its float shadow module"""
+
+        def compare_and_validate_results(
+            float_model, q_model, module_swap_list, input, hidden
+        ):
+            ob_dict = compare_model_stub_fx(
+                float_model, q_model, module_swap_list, input, hidden
+            )
+            self.assertEqual(len(ob_dict), 1)
```

Comment:
Could you add the check if the key in ob_dict is expected?

Reply:
Added in compare_model_stub PR